android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java

```diff
@@ -3765,6 +3795,47 @@ private static void addBookmark(String title, Integer pageNumber, MethodChannel.
         }
     }
 
+    private static void getScrollPos(MethodChannel.Result result, ViewerComponent component) throws JSONException {
+        PDFViewCtrl pdfViewCtrl = component.getPdfViewCtrl();
+        JSONObject jsonObject = new JSONObject();
+
+        if (pdfViewCtrl == null) {
+            result.error("InvalidState", "PDFViewCtrl not found", null);
+            return;
+        }
+
+        jsonObject.put(SCROLL_HORIZONTAL, pdfViewCtrl.getHScrollPos());
+        jsonObject.put(SCROLL_VERTICAL, pdfViewCtrl.getVScrollPos());
+
+        result.success(jsonObject.toString());
+    }
+
+    private static void setHorizontalScrollPosition(MethodCall call, MethodChannel.Result result, ViewerComponent component) {
+        PDFViewCtrl pdfViewCtrl = component.getPdfViewCtrl();
+        int horizontalScrollPosition = call.argument(KEY_HORIZONTAL_SCROLL_POSITION);
+        
+        if (pdfViewCtrl == null) {
+            result.error("InvalidState", "PDFViewCtrl not found", null);
+            return;
+        }
+
+        pdfViewCtrl.setHScrollPos((int) (horizontalScrollPosition + 0.5));
```

Comment:
unsure why the 0.5 is needed, original value is int, the method expects int as well